CMakeLists.txt

```diff
@@ -538,8 +538,15 @@ include_directories(SYSTEM velox/external)
 
 # these were previously vendored in third-party/
 if(NOT VELOX_DISABLE_GOOGLETEST)
-  set(gtest_SOURCE AUTO)
-  resolve_dependency(gtest)
+  # NOTE: Use "GTest" other than "gtest" or "GTEST" because both the
+  # "FindGTest.cmake" (See [1]) provided by cmake and the cmake file generated
+  # when compiling and installing gtest from source, "GTest" is used, not
+  # "gtest" or "GTEST".
+  #
+  # [1]
+  # https://github.com/Kitware/CMake/blob/8fe9454bcca64a327fb3e0ccf5aff7477eb036cc/Modules/FindGTest.cmake#L19-L23
```

Comment:
```suggestion
```
Thanks for the explanation but I don't think we need to keep the comment.

Reply:
Removed, thanks.